guide/color-palette.html

```diff
@@ -254,17 +254,18 @@
             <li class="nav-item">
               <a class="nav-link" href="index.html">Home</a>
             </li>
-            <li class="nav-item">
+            <li class="nav-item active">

```

Comment:
The "active" class should stay on the color-palette line li rather than getting copied over to the component one. Then again, it appears with the new subnav, the class could be removed altogether.